sdk-wallet/user_secret_key.md

```diff
@@ -1,3 +1,5 @@
+## UserSecretKey
```

Comment:
Not done, Sign and Verify functions should disappear from the private/public key definitions

Reply:
Fixed in the separate PR for sdk-wallet, now brought here.